packages/files/_test.pony

```diff
@@ -189,10 +194,37 @@ class iso _TestSymlink is UnitTest
       target_path.symlink(link_path),
       "could not create symbolic link to: " + target_path.path)
 
+
+class iso _TestFilePathFrom is UnitTest
+  fun name(): String => "files/FilePath.from-success"
+  fun apply(h: TestHelper) =>
+    try
+      let path = "tmp.filepath"
+      let ambient = h.env.root as AmbientAuth
+      let filepath = FilePath(FileAuth(ambient), path)
+      h.assert_no_error({()? => FilePath.from(filepath, path)? })
+    else
+      h.fail("Unhandled error!")
```

Comment:
i think a more informative fail message would be good here. ok, we got an error, why might that be?